SYCL/ESIMD/api/functional/value.hpp

```diff
@@ -59,6 +59,27 @@ sycl::half half_from_bytes(unsigned char hi, unsigned char lo) {
   return esimd_test::bit_cast<sycl::half>(out);
 }
 
+// Constructs std::vector with from provided vectors.
+template <typename DataT, int NumElems>
+std::vector<DataT> construct_ref_data(double step,
+                                      std::vector<DataT> first_vector,
+                                      std::vector<DataT> second_vector = {}) {
```

Comment:
```suggestion
                                      std::vector<DataT>&& first_vector,
                                      const std::vector<DataT>& second_vector = {}) {
```
Looks like you are using rvalues anyway, so you might use move constructor for the `first_vector` part.

Reply:
I've done it [here](https://github.com/intel/llvm-test-suite/pull/696/commits/c9bc321af2b1bf1b9c70b323c48041dd1e79a85e#diff-f085fe3f94e6fdb4bc2da0518bd492beb702cd3f8181a461be78bcc561ccc514L64-R66)